### ISSUE TITLE
habridge id different in openhab then specified in habridge #22

### ISSUE THREAD

**Issue by GravityRZ** [bwssytems/ha-bridge#1104]
trying to add habridge to openhab.
i get a failed to create new user error
![image](https://user-images.githubusercontent.com/16853535/59584084-69e07280-90dd-11e9-8247-7ec6abd51418.png)

the Hue hub id in openhab is way lower then current versions
it should be 00178820e229
i have put this number in the emulate hub filed but it does not seem to pass it along to openhab(or any other app)

when i also put this number in the mac field it seems to appear multiple times in openhab with the correct number but discovery still seems strange and not trusted

still  with the right id i get the failed user error

i think the serialnumber is not parsed correctly

**Comment by bwssytems** [comment]
You will need to activate the use of the link button in the ha-bridge to work with openHab. Go to the security settings in the bridge control tab and select "use link button". This will add it to the bridge devices tab. When you add the "Thing" to openHab after it has been found and it is in the "Inbox" Select the check mark and then on the ha-bridge UI, select the link button and then you are good to go.

**Comment by GravityRZ** [comment]
tried it but still not working.
in some cases i get the statement from openhab to press the pair button but when i press the link button in habridge i still get the Failed to create new user on hue bridge.

i think it is related to the sn in habridge.
this is what i get when i do not put a sn or mac adres in habridge
real bridge on address 40, habridge on address 50(note the old sn)

when i put in the new sn in habridge still the same output
![image](https://user-images.githubusercontent.com/16853535/59631257-acd73000-9147-11e9-91a0-1359fbed4c91.png)

when i put the new sn in both the emulate sn field as well as the emulate mac address field i get this
is see ip 40 flashing by multiple time but the end screen is like this
![image](https://user-images.githubusercontent.com/16853535/59631650-a1383900-9148-11e9-8b77-f639500b98dd.png)


so basically with the old sn things are not working at all(bug in the emulate sn field)
when i put in trhe new sn in both emulate sn/mac address things are going 1 step further but then i think things get mized up

**Comment by bwssytems** [comment]
You should disconnect your real hue bridge while you pair with the ha-bridge. Also, I just tested this with openhab and works fine.

**Comment by GravityRZ** [comment]
Ok will try this but it still seems the habridge hub id parsing does not work

**Comment by GravityRZ** [comment]
i must have al the bad luck in the world. it simply is not working.
switched the real hub off
do a discover, press add thing andf it instantly ads it withe the failed user mesage. no messsage regarding pressing the link button or whatever.
this is what is in the habridge settings
![image](https://user-images.githubusercontent.com/16853535/59694220-8f0dd780-91e8-11e9-94fc-81224848cf65.png)
![image](https://user-images.githubusercontent.com/16853535/59694259-9e8d2080-91e8-11e9-8425-ed549316a2b7.png)

![image](https://user-images.githubusercontent.com/16853535/59694318-b664a480-91e8-11e9-8807-3e9edf1f9061.png)

without the sn in the sn/mac field it is still not working(old sn is displayed)

**Comment by GravityRZ** [comment]
i think the problem is in openhab v2.4.0
installed openhab v 2.5.0(snapshot version)  and it works as you told me.
